sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "CSL Overawards Balance (FT)",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''CSLP''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "CSL Overawards Balance (PT)",
+            (
+                select
+                    SUM (disbursement_overawards.overaward_value)
+                from
+                    sims.disbursement_overawards disbursement_overawards
+                where
+                    disbursement_overawards.student_id = students.id
+                    and disbursement_overawards.disbursement_value_code = ''BCSL''
+                GROUP by
+                    disbursement_overawards.student_id,
+                    disbursement_overawards.disbursement_value_code
+            ) as "BCSL Overawards Balance",
+            students.disability_status as "Permanent Disability status",
+            -- Application Personal Info (does not include Institution or Program info)
+            ''"'' || program_years.program_year_desc || ''"'' as "Program Year",
+            applications.data ->> ''studentNumber'' as "Student Number",
+            applications.data ->> ''citizenship'' as "Citizenship status",
+            applications.data ->> ''bcResident'' as "BC Residency Status",
+            applications.data ->> ''indigenousStatus'' as "Indigenous Person status",
+            applications.data ->> ''aboriginalHeritage'' as "Indigenous category code",
+            applications.data ->> ''applicationPDPPDStatus'' as "PD/PPD Application Flag",
+            applications.data ->> ''youthInCare'' as "Youth in Care Flag",
+            ''--TBD--'' as "Youth in Care beyond age 19 Flag",
+            applications.data ->> ''outOfHighSchoolFor4Years'' as "Outside of highschool >4 years flag",
+            applications.data ->> ''whenDidYouGraduateOrLeaveHighSchool'' as "High School Graduation Date",
+            applications.data ->> ''fulltimelabourForce'' as "Recent Labour Force Participation Flag",
+            applications.data ->> ''addTrustContactToContactSABC'' as "Trusted Contact Identified (Yes/No)",
+            applications.data ->> ''relationshipStatus'' as "Marital Status",
+            --Application Financial Info
+            applications.data ->> ''taxReturnIncome'' as "Total income",
+            applications.data ->> ''partTimeAwardTypesToBeConsidered'' as "Grant-only flag",
```

Comment:
From the document I understood this flag to be "Y" | "N". But if business is good with the fromio option values, it is ok.

Reply:
That can be changed in the next ticket if the business require it.